File: packages/pensopay_android/android/src/main/kotlin/com/flutter/pensopay/networking/pensopayapi/pensopaylink/models/PPOperation.kt
package com.flutter.pensopay.networking.pensopayapi.pensopaylink.models

import org.json.JSONObject

class PPOperation(id: Int): JSONObject() {

    // Required Properties

    var id: Int = id


    // Optional Properties

    var type: String? = null
    var amount: Int? = null
    var pending: Boolean? = null
    var pp_status_code: String? = null
    var pp_status_msg: String? = null
    var aq_status_msg: String? = null
    var aqStatusMsg: String? = null
    var data: Map<String, String>? = null
    var callback_url: String? = null
    var callback_success: Boolean? = null
    var callback_response_code: Int? = null
    var callback_duration: Int? = null
    var acquirer: String? = null
    var callback_at: String? = null
    var created_at: String? = null

}

File: packages/pensopay_ios/ios/Classes/Networking/Pensopay API/Pensopay Link/Models/PPOperation.swift
//
//  PPOperation.swift
//  PensopaySDK
//
//  Created on 07/26/2022
//  Copyright © 2022 Pensopay. All rights reserved.
//

import Foundation

public class PPOperation : Codable {
    
    // MARK: - Properties
    
    public var id: Int
    public var type: String?
    public var amount: Int?
    public var pending: Bool?
    public var PP_status_code: String?
    public var PP_status_msg: String?
    public var aq_status_msg: String?
    public var aqStatusMsg: String?
    public var data: Dictionary<String, String>?
    public var callback_url: String?
    public var callback_success: Bool?
    public var callback_response_code: Int?
    public var callback_duration: Int?
    public var acquirer: String?
    public var callback_at: String?
    public var created_at: String?
    
}
